Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,94 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser .
+
+== Stack specific solution
+A stack specific solution is the place where you can find all the solutions of each stack. 
+You should do each solution on a new branch and not take the master branch because that can lead to conflicts.
+
+
+== Fetch upstream
```

Comment:
You should tell the user that you have to fork OR to fetch all, depending on whether the repo already exists